lib/util/git/types.ts

```diff
@@ -75,6 +75,22 @@ export interface CommitFilesConfig {
   force?: boolean;
 }
 
+type BranchName = string;
+type TargetBranchName = BranchName;
+type SourceBranchName = BranchName;
+
+export type GitConflictsCache = Record<TargetBranchName, TargetBranchConflicts>;
+
+interface TargetBranchConflicts {
```

Comment:
```suggestion
export interface TargetBranchConflicts {
```
We should export public interfaces when they are already exported implicit